nucypher/cli/painting/status.py

```diff
@@ -167,10 +167,13 @@ def paint_stakers(emitter, stakers: List[str], registry: BaseContractRegistry) -
         owned_in_nu = round(owned_tokens, 2)
         current_locked_tokens = round(staker.locked_tokens(periods=0), 2)
         next_locked_tokens = round(staker.locked_tokens(periods=1), 2)
+        reward_amount = NU.from_nunits(staking_agent.calculate_staking_reward(staker_address=staker_address))
 
         emitter.echo(f"{tab}  {'Owned:':10} {owned_in_nu}")
         emitter.echo(f"{tab}  Staked in current period: {current_locked_tokens}")
         emitter.echo(f"{tab}  Staked in next period: {next_locked_tokens}")
+        emitter.echo(f"{tab}  Unlocked: {reward_amount}")
```

Comment:
If using `reward_amount`, we should label `Unlocked rewards`, but if the goal is to show unlocked tokens, then would the calculation instead need to be `owned_in_nu - current_locked_tokens`?

Reply:
✔️